pyensembl/sequence_data.py

```diff
@@ -36,22 +38,29 @@ class SequenceData(object):
     """
     def __init__(
             self,
-            fasta_path,
+            fasta_paths,
             require_ensembl_ids=False,
             cache_directory_path=None):
-        self.fasta_path = abspath(fasta_path)
-        self.fasta_directory_path, self.fasta_filename = split(self.fasta_path)
+
+        if isinstance(fasta_paths, string_types):
+            fasta_paths = [fasta_paths]
+
+        self.fasta_paths = [abspath(path) for path in fasta_paths]
+        self.fasta_directory_paths = [split(path)[0] for path in self.fasta_paths]
+        self.fasta_filenames = [split(path)[1] for path in self.fasta_paths]
         if cache_directory_path:
-            self.cache_directory_path = cache_directory_path
+            self.cache_directory_paths = [cache_directory_path]*len(self.fasta_paths)
```

Comment:
I will fix this. I do think it would be a cleaner solution to just put all files into one cache directory. 

It seems the user cannot specify the cache directory (unless they are interfacing with the API directly in a script), so it would need to be automatically determined. But the way things are done currently, the cache directory (if it not supplied through `cache_directory_path `) is determined by the paths of the supplied FASTA files. And the case may arise where the user provides multiple transcript or protein FASTA files where each is located in separate directories, so that I why I used a list for `cache_directory_paths`.